src/transformers/modeling_utils.py

```diff
@@ -4303,7 +4308,7 @@ def from_pretrained(
                 dispatch_model(model, **device_map_kwargs)
 
         if hf_quantizer is not None:
-            hf_quantizer.postprocess_model(model)
+            hf_quantizer.postprocess_model(model, config=config)
```

Comment:
Since there's no cases in which config should not be passed, we should put this directly in the function signature, rather than implicitly in the kwargs

Reply:
config is only needed for compressed tensors hfquantizer to get the names_or_path. 
I dont think its needed to make it default since no other hfquantizer is using it